client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -90,6 +92,8 @@ export const PrescriptionLineEditForm: React.FC<
   programId,
 }) => {
   const t = useTranslation();
+  const p = Pluralize;
```

Comment:
No need to rename here -- because it's not a named export, you can just do `import p from Pluralize` (but this should be in the IntlUtils module anyway)